pegen/pegen.c

```diff
@@ -505,22 +505,22 @@ seq_flatten(Parser *p, asdl_seq *seqs)
 }
 
 expr_ty
-seq_to_dotted_name(Parser *p, asdl_seq *seq)
+join_names_with_dot(Parser *p, expr_ty first_name, expr_ty second_name)
 {
-    /* Create a string of the form a.b.c like Python/ast.c alias_for_dotted_name does
-       for dotted names */
-    if (asdl_seq_LEN(seq) == 1) {
-        return asdl_seq_GET(seq, 0);
-    }
+    /* Create a string of the form "first_name.second_name" like
+       Python/ast.c alias_for_dotted_name does for dotted names */
```

Comment:
This comment doesn't add a lot of value (and eventually there won't be a Python/ast.c :-).